recipes/tetools_l-modolo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.0.0" %}
+{% set sha256 = "b884fce471e90b35b9be898e68095379f84cc2d807054667c3daaaa8f46ec30c" %}
+
+package:
+  name: tetools_l-modolo
+  version: {{ version }}
+
+source:
+  url: https://github.com/l-modolo/TEtools/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True # [osx]
```

Comment:
Needed? This looks no-arch to me, actually. No binaries, so should work on OSX, too?